packages/fast-components-react-msft/src/media/examples.data.ts

```diff
@@ -0,0 +1,18 @@
+import {ICategoryItemProps} from "@microsoft/fast-development-site-react";
+import {IGenericExample} from "../examples";
+import {IManagedClasses} from "@microsoft/fast-jss-manager-react";
+import Media from "./index";
+import {IMediaProps} from "@microsoft/fast-components-react-base";
+
+export default {
+    name: "media",
+    component: Media,
+    data: [
+        {
+            alt: "Alt tag",
```

Comment:
https, like stated above - also, for alt tags we usually go with something like: "Placeholder with grey background and dimension watermark without any imagery"